.github/workflows/docker-hub-latest.yml

```diff
@@ -0,0 +1,40 @@
+# Based on https://github.com/matrix-org/dendrite/blob/master/.github/workflows/docker-hub.yml
+
+name: "Docker Hub - Latest"
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+
+env:
+  DOCKER_NAMESPACE: matrixdotorg
+  PLATFORMS: linux/amd64,linux/arm64
+  # Only push if this is develop, otherwise we just want to build
+  # On a PR github.ref is the target branch, so don't push for that either
+  PUSH: ${{ github.ref == 'refs/heads/master' && github.event_name != 'pull_request' }}
+
+jobs:
+  docker-latest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        if: ${{ env.PUSH == 'true' }}
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
```

Comment:
You're probably going to have to ask SRE/IT to share the secret with this repo, if it isn't already

Reply:
Doesn't look like it's shared, from https://github.com/matrix-org/matrixmon/settings/secrets/actions

You could also upload to ghcr.io if that's any easier.

Reply:
Unsure whether to maintain the status quo or ghcr.io. Given how little this image is updated, we might as well go with ghcr.io and tell people to migrate?

Reply:
Oh, I didn't realise there was an existing image on dockerhub for this. Would be nice to keep that updated if we can... but no strong opinions from me either way.

Reply:
I've agonized over this one for a day now and I'm really unsure. ghcr.io is convenient from a configuration point of view but it's a breaking change....

I think I'm going to revert what I did and go back to dockerhub. Instead, I'll ask ops for some lovely tokens.